addons/binding/org.openhab.binding.energenie/src/main/java/org/openhab/binding/energenie/internal/api/manager/EnergenieApiManager.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.energenie.internal.api.manager;
+
+import org.openhab.binding.energenie.internal.api.EnergenieDeviceTypes;
+import org.openhab.binding.energenie.internal.api.JsonDevice;
+import org.openhab.binding.energenie.internal.api.JsonGateway;
+import org.openhab.binding.energenie.internal.api.JsonSubdevice;
+
+/**
+ * A helper class for executing requests to the Mi|Home REST API.
+ * For more information see {@link https://mihome4u.co.uk/docs/api-documentation}
+ *
+ * @author Svilen Valkanov - Initial contribution
+ * @author Dimitar Ivanov - Initial contribution and API
```

Comment:
If you both did the initial contribution, I would at least expect exactly the same line there.